crates/transaction-emitter-lib/src/emitter/mod.rs

```diff
@@ -439,70 +552,121 @@ async fn wait_for_single_account_sequence(
 /// will be, since not all of the submitted transactions were accepted.
 /// TODO, investigate whether this behaviour is desirable.
 async fn wait_for_accounts_sequence(
+    start_time: Instant,
     client: &RestClient,
     accounts: &mut [LocalAccount],
+    transactions_per_account: usize,
     wait_timeout: Duration,
+    fetch_only_once: bool,
```

Comment:
adding this mode, for the ConstTPS, so that we fetch accounts only once, instead of looping.